src/NuGet.CommandLine.StagingExtensions/StagingResources.resx

```diff
@@ -198,4 +210,52 @@ stage -drop 4b139cb7-c4d4-4541-8c05-0f41ba5ab945 -source https://api.nuget.org/v
   <data name="StagingNotSupported" xml:space="preserve">
     <value>The specified server does not support staging.</value>
   </data>
+  <data name="StageCommandGetDescription" xml:space="preserve">
+    <value>Get stage details</value>
+  </data>
+  <data name="None" xml:space="preserve">
+    <value>None</value>
+  </data>
+  <data name="StageCommandCommitDescription" xml:space="preserve">
+    <value>Commit staged packages to source feed, and remove stage</value>
+  </data>
+  <data name="StageCommandProgressDescription" xml:space="preserve">
+    <value>Get status of commit progress</value>
+  </data>
+  <data name="CommitCommandWasCanceled" xml:space="preserve">
+    <value>Commit was canceled</value>
+  </data>
+  <data name="ConfirmStageCommit" xml:space="preserve">
+    <value>Stage '{0}' will be commited. Do you want to continue?</value>
+  </data>
+  <data name="CommittingStageMessage" xml:space="preserve">
+    <value>Committing stage with id '{0}'</value>
+  </data>
+  <data name="Completed" xml:space="preserve">
+    <value>Completed</value>
+  </data>
+  <data name="Failed" xml:space="preserve">
+    <value>Failed</value>
+  </data>
+  <data name="InProgress" xml:space="preserve">
+    <value>In progress</value>
+  </data>
+  <data name="Pending" xml:space="preserve">
+    <value>Pending</value>
+  </data>
+  <data name="RetryCommitMessage" xml:space="preserve">
+    <value>Please retry commit or contact support in case of recuring failures</value>

```

Comment:
Thanks :)

---
In reply to: [78237097](https://github.com/NuGet/NuGet.Staging/pull/73#discussion_r78237097) [](ancestors = 78237097)